clang/lib/CIR/CodeGen/CIRGenItaniumCXXABI.cpp

```diff
@@ -1432,6 +1433,10 @@ void CIRGenItaniumRTTIBuilder::BuildSIClassTypeInfo(mlir::Location loc,
   Fields.push_back(BaseTypeInfo);
 }
 
+void CIRGenItaniumRTTIBuilder::BuildVMIClassTypeInfo(const CXXRecordDecl *RD) {
+  // TODO: Implement this function.
```

Comment:
Is this the RTTI stuff you said it's coming next, right?

Reply:
Yeah.